ui/statistics_button.js

```diff
@@ -81,48 +81,51 @@ shaka.ui.StatisticsButton = class extends shaka.ui.Element {
     /** @private {!Object.<string, HTMLElement>} */
     this.displayedElements_ = {};
 
-    /** @private {!Object.<string, string>} */
-    this.parseFrom_ = {
-      'width': 'px',
-      'height': 'px',
-      'completionPercent': 'percent',
-      'bufferingTime': 'seconds',
-      'drmTimeSeconds': 'seconds',
-      'licenseTime': 'seconds',
-      'liveLatency': 'seconds',
-      'loadLatency': 'seconds',
-      'manifestTimeSeconds': 'seconds',
-      'estimatedBandwidth': 'bits',
-      'streamBandwidth': 'bits',
-      'maxSegmentDuration': 'time',
-      'pauseTime': 'time',
-      'playTime': 'time',
-      'corruptedFrames': 'frames',
-      'decodedFrames': 'frames',
-      'droppedFrames': 'frames',
+
+    const parsePx = (name) => {
+      return this.currentStats_[name] + ' (px)';
+    };
+
+    const parsePercent = (name) => {
+      return this.currentStats_[name] + ' (%)';
+    };
+
+    const parseFrames = (name) => {
+      return this.currentStats_[name] + ' (frames)';
+    };
+
+    const parseSeconds = (name) => {
+      return this.currentStats_[name].toFixed(2) + ' (s)';
     };
 
-    /** @private {!Object.<string, Function>} */
-    this.parseTo_ = {
-      'px': (name) => {
-        return this.currentStats_[name] + ' (px)';
-      },
-      'percent': (name) => {
-        return this.currentStats_[name] + ' (%)';
-      },
-      'frames': (name) => {
-        return this.currentStats_[name] + ' (frames)';
-      },
-      'seconds': (name) => {
-        return this.currentStats_[name].toFixed(2) + ' (s)';
-      },
-      'bits': (name) => {
-        return Math.round(this.currentStats_[name] / 1000) + ' (kbits/s)';
-      },
-      'time': (name) => {
-        return shaka.ui.Utils.buildTimeString(
-            this.currentStats_[name], false) + ' (m)';
-      },
+    const parseBits = (name) => {
+      return Math.round(this.currentStats_[name] / 1000) + ' (kbits/s)';
+    };
+
+    const parseTime = (name) => {
+      return shaka.ui.Utils.buildTimeString(
+          this.currentStats_[name], false) + ' (m)';
+    };
+
+    /** @private {!Object.<string, function(string)>} */
```

Comment:
Technically, the functions are all function(string):string

Reply:
Thank you Theodab, I grouped all the improvements on my last commit!

Reply:
Ah, sorry about that, I had meant to start a review but I accidentally posted those all as individual comments, and I only noticed when I got to the last one. I was a bit worried that all of my edits would make the emails confusing.